Obsidian/MainWindow.xaml.cs

```diff
@@ -30,11 +30,11 @@ namespace Obsidian
     /// </summary>
     public partial class MainWindow
     {
-        public Dictionary<string, object> Config { get; }
+        private readonly Dictionary<string, object> Config;
         private static readonly ILog Logger = LogManager.GetLogger("MainWindow");
-        public WADFile Wad { get; set; }
-        public WADEntry CurrentlySelectedEntry { get; set; }
-        public static Dictionary<ulong, string> StringDictionary { get; set; } = new Dictionary<ulong, string>();
+        private WADFile Wad;
+        private WADEntry CurrentlySelectedEntry;
+        public static Dictionary<ulong, string> StringDictionary { get; private set; } = new Dictionary<ulong, string>();
```

Comment:
revert

Reply:
Reasoning for having it public instead of private?

Reply:
so it can be accessed from other classes

Reply:
I'd argue that the main decision should be in whether other classes *should* be able to access it or not. Judging from the fact currently no other class uses these properties, it seems to me that it's unnecessary to have them public when they could be private instead.

Reply:
sorry thought this was changed to private too, private set is fine